cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/telemetry/ui_metric.spec.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { MiscUtils } from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+import { CURRENT_TENANT } from '../../../../../utils/commands';
+
+const miscUtils = new MiscUtils(cy);
+
+describe('dev_console_ui_metric', () => {
+  before(() => {
+    CURRENT_TENANT.newTenant = 'global';
+    miscUtils.visitPage('app/dev_tools#/console');
+    cy.wait(30000);
+    cy.get('[data-test-subj="help-close-button"]').then(($btn) => {
+      if ($btn.is(':visible')) {
+        cy.wrap($btn).click({ force: true });
+      } else {
+        cy.get('[type="button"]').contains('Console').click({ force: true });
+      }
+    });
+    cy.wait(5000);
+  });
+
+  it('check UI Metric are being recorded', function () {
+    cy.intercept('api/ui_metric/report', cy.spy().as('reportreq'));
+
+    miscUtils.visitPage('app/home#/');
+
+    cy.wait(45000);
+
+    cy.get('@reportreq').should('be.called');
+
+    // Now verify the response of api/stat
+
+    cy.wait(5000);
+
+    cy.request(
+      'GET',
+      'http://localhost:5601/api/stats?extended=true&legacy=true&exclude_usage=false'
```

Comment:
pull from cypress settings the dashboards url.

Reply:
https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/cypress.json#L5

Reply:
@kavilla Update to the use base path from config.